pkg/kubectl/cmd/taint.go

```diff
@@ -72,7 +73,17 @@ func NewCmdTaint(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 
 	// retrieve a list of handled resources from printer as valid args
 	validArgs := []string{}
-	p, err := f.Printer(nil, false, false, false, false, false, false, []string{})
+	p, err := f.Printer(nil, &kubectl.PrintOptions{
```

Comment:
@janetkuo 
> Suggest just use &kubectl.PrintOptions{} here (and everywhere else that applies) since you're setting zero values here?

Thanks for the feedback, Done!